content/sensu-core/2.0/reference/agent.md

```diff
@@ -0,0 +1,822 @@
+{{< /highlight >}}
+
+To see available configuration flags and defaults:
+
+{{< highlight shell >}}
+sensu-agent start --help
+{{< /highlight >}}
+
+If no configuration flags are provided, the agent loads configuration from [`/etc/sensu/agent.yml`][25] by default.
+
+To start the agent using a service manager:
+
+{{< platformBlock "Linux" >}}
+
+**Linux**
+
+{{< highlight shell >}}
+service sensu-agent start
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+**Windows**
+
+{{< highlight shell >}}
+sc start sensu-agent
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+### Stopping the service
+
+To stop the agent service using a service manager:
+
+{{< platformBlock "Linux" >}}
+
+**Linux**
+
+{{< highlight shell >}}
+service sensu-agent stop
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+**Windows**
+
+{{< highlight shell >}}
+sc stop sensu-agent
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+### Restarting the service
+
+You must restart the agent to implement any configuration updates.
+
+To restart the agent using a service manager:
+
+{{< platformBlock "Linux" >}}
+
+**Linux**
+
+{{< highlight shell >}}
+service sensu-agent restart
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+**Windows**
+
+{{< highlight shell >}}
+sc restart sensu-agent
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+### Enabling on boot
+
+To enable the agent to start on system boot:
+
+{{< platformBlock "Linux" >}}
+
+**Linux**
+
+{{< highlight shell >}}
+systemctl enable sensu-agent
+{{< /highlight >}}
+
+To disable the agent from starting on system boot:
+
+{{< highlight shell >}}
+systemctl disable sensu-agent
+{{< /highlight >}}
+
+_NOTE: On older distributions of Linux, use `sudo chkconfig sensu-server on` to enable the agent and `sudo chkconfig sensu-server off` to disable._
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+**Windows**
+
+{{< highlight shell >}}
+sc config sensu-agent start=auto
+{{< /highlight >}}
+
+To disable the agent from starting on system boot:
+
+{{< highlight shell >}}
+sc config sensu-agent start=disabled
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+### Getting service status
+
+To see the status of the agent service using a service manager:
+
+{{< platformBlock "Linux" >}}
+
+**Linux**
+
+{{< highlight shell >}}
+service sensu-agent status
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+{{< platformBlock "Windows" >}}
+
+**Windows**
+{{< highlight shell >}}
+sc query sensu-agent
+{{< /highlight >}}
+
+{{< platformBlockClose >}}
+
+### Getting service version
+
+To get the current agent version using the `sensu-agent` tool:
+
+{{< highlight shell >}}
+sensu-agent version
+{{< /highlight >}}
+
+### Getting help
+
+The `sensu-agent` tool provides general and command-specific help flags:
+
+{{< highlight shell >}}
+# Show sensu-agent commands
+sensu-agent help
```

Comment:
I just realized this command actually starts the agent (there's only one command so I suppose it just ignores the start command). We should probably do `sensu-agent --help` instead.